lib/hub-registry.js

```diff
@@ -126,18 +103,9 @@ HubRegistry.prototype.init = function (taker) {
  * @param {function} fn - the task function
  */
 HubRegistry.prototype.set = function (name, fn) {
+	// TODO: what if this collides with a master task?
+	ForwardRegistry.prototype.set.call(this, name, fn);
```

Comment:
I think it is not necessary to call ForwardRegistry.set here as the tasks will all be recreated during init. That is I don't think it is possible for HubRegistry.set to be called after HubRegistry.init has run? I'll take a look at it after the merge...